packages/headless/src/features/context/context-state.ts

```diff
@@ -1,6 +1,18 @@
+/**
+ * Represents a contextual value, which can be a string or an array of string.
+ */
 export type ContextValue = string | string[];
+/**
+ * Represents a key/value pair of contextual information.
+ */
 export type Context = Record<string, ContextValue>;
-export type ContextState = {contextValues: Context};
+export type ContextState = {
+  /**
+   * Hold the contextual information that can be leveraged by the Coveo platform to provide relevant results.
+   * {@link https://docs.coveo.com/en/2081/coveo-machine-learning/understanding-custom-context}
```

Comment:
How does this way of embedding links compare to the markdown syntax? e.g. `[leveraged by the Coveo platform](<link>)`

I believe the doc team prefers the markdown approach.

Reply:
👍